app/models/person.rb

```diff
@@ -16,4 +16,8 @@ def home_address?
   def correspondence_address?
     correspondence_address&.address_line_one.present?
   end
+
+  def has_passporting_benefit?
+    BenefitType.passporting.map(&:value).include?(benefit_type&.to_sym)
+  end
```

Comment:
Moved from applicant to person as both applicant and partner will use this method

Reply:
It would be good to update the method in TypeOfMeansAssessent to use this method and be partner aware.

Reply:
Will do this in the next ticket i've picked up or with the task list updates